internal/plots/benchmark_architecture_os_comparision.dat

```diff
@@ -1,8 +1,8 @@
-# System                        OS          Latency Same Core       Latency Different Cores
-  "Raspberry Pi 4B"             "Linux"     824                     0
-  "Intel i7-13700H"             "Linux"     92                      326
-  "Intel i7-8565U"              "Linux"     270                     263
-  "Intel i7-7500U"              "Linux"     276
-  "AMD Ryzen 7 7840S"           "Linux"     109                     222
-  "AMD Ryzen 9 5950X"           "Windows"   205
-  "AMD Ryzen 9 5950X"           "Linux"     168                     173
+# System                        OS          Latency Same Core (HyperThreading)      Latency Different Cores
+  "Raspberry Pi 4B"             "Linux"     774                                     0
+  "Intel i7-13700H"             "Linux"     92                                      326
+  "Intel i7-8565U"              "Linux"     270                                     263
+  "Intel i7-7500U"              "Linux"     276                                     0
+  "AMD Ryzen 7 7840S"           "Linux"     109                                     222
+  "AMD Ryzen 9 5950X"           "Windows"   205                                     0
+  "AMD Ryzen 9 5950X"           "Linux"     168                                     173
```

Comment:
Have you tried something like this?
```suggestion
  "AMD Ryzen 9 5950X"           "Linux [Intra-Core]"     168
  "AMD Ryzen 9 5950X"           "Linux [Inter-Core]"     173
```

Reply:
Yes, but it is for me not directly obvious what inter and intra core means. So we need an explanation inside the diagram with intra-core means hyperthreading and one core and inter core means different physical cpu cores but on the same CPU.

The intention of this PR for the moment was to just update the numbers. We can add this before the v0.4 release but I would do this in a separate PR if you do not mind.

Reply:
No problem, we can also just keep it as is.